packages/field/src/react/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default } from './field'
 export * from './field'
+export { FieldInputProps } from './input'
```

Comment:
Why was this change required?

Reply:
That type is useful when composing components using field so I'm exposing it as that's the purpose of field.

Reply:
We often accomplish this through: `React.ComponentProps<typeof FieldInput>`. 

But we aren't consistent and sometimes export the specific prop interface too. Preference?

Reply:
I prefer interfaces. I've found this actually contributed to the css prop error in #1698

Reply:
orly... well, that'd be a great reason not to use React.ComponentProps. Look at the #1698 PR though, I don't see it used; I'm missing the connection there. Mind specifying that for me?

Reply:
https://github.com/pluralsight/design-system/pull/1698/files#r648525051